### ISSUE TITLE
Newcomers spawn and leave at edge of town 

### ISSUE THREAD

**Issue by carlansor** [ldtteam/minecolonies-features#126]
# Prerequisites

- [ ] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [ ] I checked the MineColonies [Wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my feature is not covered there.
- [ ] I made sure that **this issue is not a duplicate of any existing issue.**
- [X] This is a feature request issue and not a bug report. Please create all bug reports [here](https://github.com/ldtteam/minecolonies/issues).

<!-- Please fill out this template as best as possible, this will help
     us to understand your issue much easier. --

#### Specifications

* Citizens could spawn at the edge of the colony (similar to how barbarians and raiders spawn) and walk to the tavern. They could also leave the same way around.

Expansion:
 In future updates these newcomers could visit other buildings such as the restaurant (maybe eat something and pay for it), the townhall for diplomatic request, or any crafter hut to trade. Some could come with malicious intents as robery, assasination or spy. And guards could kick them out

# Viewers

* Add a thumbs-up to the feature request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Optionally add a comment with any insight and background you have that isn't already part of the conversation.

**Comment by ravenbuilder934** [comment]
Is this like the tavern travelers? What do you mean?

**Comment by carlansor** [comment]
Yes, as thought on discord, newcomers that currently spawn in the tavern could spawn at the edge of the town and walk into the tavern. Maybe leaving the same way. Simple and nice :)

**Comment by ravenbuilder934** [comment]
Ooh, yes! Makes more sense than randomly appearing inside the tavern.

**Comment by carlansor** [comment]
in the future these newcomers could visit other huts too like the restaurant or the future marketplace.
